### ISSUE TITLE
✨ [API Mutation] Mutation to edit Work Streams

### ISSUE THREAD

**Issue by tristan-orourke** [GCTC-NTGC/gc-digital-talent#7674]
## ✨ Feature

Describe the feature that is being requested

## 🕵️ Details

Add any additional details that could assist with the development of the new feature.

## 🎨 Design File

Include a link to the design file (if it exists).

### 🧑‍🎨 Designer

Include the author of the linked design file's name (or even better, their `@githubUsername`) for future reference.

## 📸 Screenshot

Add a screenshot of the new feature (if possible).

## 🙋‍♀️ Proposed Implementation

(optional) Do you have a proposed implementation?

## 🌎 Localization

(optional) Provide any new copy along with translations available.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the feature was properly implemented.

- [ ] Criteria 1
- [ ] Criteria 2

## 🛑 Blockers
```[tasklist]
### Blocked By
- [ ] #7574
```

**Comment by mnigh** [comment]
@tristan-orourke or @brindasasi is this the mutation part of https://github.com/GCTC-NTGC/gc-digital-talent/issues/12081 and if so, can this issue be closed as not planned: duplicate/stale?

**Comment by mnigh** [comment]
> is this the mutation part of [#12081](https://github.com/GCTC-NTGC/gc-digital-talent/issues/12081) and if so, can this issue be closed as not planned: duplicate/stale?

@petertgiles same question, is this the mutation part of https://github.com/GCTC-NTGC/gc-digital-talent/issues/12081 and if so, can this issue be closed as not planned: duplicate/stale?

**Comment by esizer** [comment]
Yeah, we do have a mutation do this so I think this can be closed as completed.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/a3fece998e048d5acd7f0b874c43943b0ded5761/api/graphql/schema.graphql#L2758-L2766
